python/plugins/processing/gui/ScriptEditorDialog.py

```diff
@@ -173,9 +163,9 @@ def closeEvent(self, evt):
     def editHelp(self):
         if self.alg is None:
             if self.algType == self.SCRIPT_PYTHON:
-                alg = ScriptAlgorithm(None, str(self.editor.text()))
+                alg = ScriptAlgorithm(None, unicode(self.editor.text()))
```

Comment:
(Applies to all of the str->unicode items in this pull request as far as I can tell)